coder_sniffer/DrupalPractice/ruleset.xml

```diff
@@ -12,9 +12,15 @@
     <exclude-pattern>*</exclude-pattern>
   </rule>
 
-  <rule ref="DrupalPractice.CodeAnalysis.VariableAnalysis.UndefinedVariable">
+  <rule ref="DrupalPractice.CodeAnalysis.VariableAnalysis">
     <!-- Do not run this sniff on template files. -->
     <exclude-pattern>*.tpl.php</exclude-pattern>
+    <properties>
+      <property name="allowUnusedFunctionParameters" value="1"/>
```

Comment:
Boolean values should be specified by using the strings "true" and "false" rather than the integers 0 and 1.